src/app/server.ts

```diff
@@ -21,6 +22,9 @@ router.get('/customer/:documentNumber', findCustomerByDocumentNumber);
 router.post('/product', createProduct);
 router.put('/product/:id', updateProduct);
 
+// Orders
+router.get('/orders/get', getOrders)
```

Comment:
```suggestion
router.get('/orders', getOrders)
```

um GET no /orders já identifica que é uma listagem, dai não precisa ter o /get no path